firmware/nRF_badge/data_collector/Makefile

```diff
@@ -4,13 +4,13 @@ ifndef NRF_SDK_PATH
 $(warning NRF_SDK_PATH environment variable undefined.  Default: /usr/local/nRF51_SDK_8.1.0_b6ed55f)
 endif
 
-NRF_SDK_PATH ?= /usr/local/nRF51_SDK_8.1.0_b6ed55f
+NRF_SDK_PATH ?= /Users/bernalrojas/Desktop/HD/nrf51sdk/
```

Comment:
Please revert this line to the original (just because it's better not to have user-specific values in these files). If you want to override this value, you can simple change the environmental variable in your shell